### PR TITLE
add json and quiet flags

### PR DESCRIPTION
**Pull request by brekk**
Hey there,

I added `--json`, `--quiet` and `--help` flags. I think they're pretty straightforward to use:

```
cost-of-modules -q
```

Only logs the final output, no intermediate commentary

```
cost-of-modules -j
```

Logs the output as a json blob

```
cost-of-modules -j=2
```

passes the 2 to `JSON.stringify(x, null, 2)`

```
cost-of-modules --help
```

Prints help and does no additional work.

Please let me know if there are additional steps I need to take.

Thanks,

Brekk

### CODE REVIEW

**Comment by brekk** [comment]
whoops, seeing a rebase error, will amend shortly

File: src/index.js [hunk 1]
```diff
@@ -1,70 +1,76 @@
 #!/usr/bin/env node
 
-const helpers = require('./helpers')
+const helpers = require('./helpers')(() => {
```
Comment (brekk) [inline src/index.js]: btw, prettier seems to have formatted this IIFE very oddly

**Comment by brekk** [comment]
Hey Siddharth,

Is there anything I need to do to get this merged?

Thanks,

Brekk

**Comment by siddharthkp** [comment]
Hey, sorry I was a little busy. 

Soon 🙂

**Comment by siddharthkp** [comment]
Hey, is there any additional benefit of using an IIFE here?

**Comment by brekk** [comment]
Yeah, so the idea was that I needed to add an IIFE / closure here b/c otherwise you can't exit cleanly early. Without the closure, you can't return here: https://github.com/siddharthkp/cost-of-modules/pull/44/files#diff-1fdf421c05c1140f6d71444ea2b27638R6

**Comment by brekk** [comment]
Thanks for merging!

**Comment by siddharthkp** [comment]
@brekk Don't thank me yet! It's merged into a temporary branch, not in master yet

**Comment by brekk** [comment]
@siddharthkp Word. What's the process for getting it to master?

**Comment by siddharthkp** [comment]
I quit being lazy and test it 🙈 

soon, I promise

**Comment by brekk** [comment]
@siddharthkp Is there any additional work I can do to get this expedited? What tests are required? Can I run them?

**Comment by palashmon** [comment]
It seems that travis-ci build is failing for this commit:

https://travis-ci.org/siddharthkp/cost-of-modules/builds/255999481

Please check it once.

**Pull request merged**